pointerevents/pointerevent_attributes_mouse-manual.html

```diff
@@ -88,9 +107,10 @@
         <h1>Pointer Events pointerdown tests</h1>
         <h4>
             Test Description: This test checks the properties of mouse pointer events. Move your mouse over the black square and click on it. Then move it off the black square.
+            Test passes if the proper behavior of the events is observed.
```

Comment:
Please expand into 4 steps: click, move, click, move. Perhaps mark the iframe through border/background color and say something like "...click it again when in red..."